### PR TITLE
Dougbu/add.view.component

### PR DESCRIPTION
**Pull request by dougbu**
Add a view component to BasicViews site
- also fix minor issues in solution

### CODE REVIEW

File: test/MvcBenchmarks.InMemory/HostingStartup.cs [hunk 1]
```diff
@@ -27,10 +27,14 @@ public static Action<IServiceCollection> InitializeServices(Type startup, Action
...
-                });
+                hostingEnvironment.Initialize(
+                    applicationRoot,
+                    new WebHostOptions
+                    {
+                        Environment = "Production",
+                    },
+                    configuration: null);
+
                 services.AddSingleton<IHostingEnvironment>(new HostingEnvironment());
```
Comment (dougbu) [inline test/MvcBenchmarks.InMemory/HostingStartup.cs]: If this is intentional, why is there a `hostingEnvironment` variable?

File: .gitignore [hunk 2]
```diff
@@ -34,3 +34,4 @@ nuget.exe
 *.sln.ide
 node_modules
 **/[Cc]ompiler/[Rr]esources/**/*.js
+launchSettings.json
```
Comment (rynowak) [inline .gitignore]: :dancer:

File: test/MvcBenchmarks.InMemory/HostingStartup.cs (hunk 1 above)
Comment (rynowak) [inline test/MvcBenchmarks.InMemory/HostingStartup.cs]: No idea? bug? I haven't run these tests in about 3 months.

**Comment by rynowak** [comment]
:shipit: - btw you have commit permissions on this repo.

**Comment by dougbu** [comment]
- a3f8147 for smallish updates
- c0bf2e3

**Pull request closed without merging**